gui/src/main/java/org/verapdf/cli/VeraPdfCli.java

```diff
@@ -146,7 +146,7 @@ private static void showVersionInfo(final boolean isVerbose) {
 	private static void showUpdateInfo(final ReleaseDetails details) {
 		SoftwareUpdater updater = Applications.softwareUpdater();
 		if (!updater.isOnline()) {
-			logger.log(Level.WARNING, "Update Service not online"); //$NON-NLS-1$
+			logger.log(Level.WARNING, "Update Service not available"); //$NON-NLS-1$
```

Comment:
This could use the constant in SoftwareUpdater and the other output strings could be factored out.